### ISSUE TITLE
Ayom Ruei Ayom.md

### ISSUE THREAD

**Issue by gibertgibs** [ASKnet-Open-Training/ASKnet-Track-II-participants-stories-Community-Empowerment-2020-2021#29]

**Comment by wunbaai** [comment]
Ayom Ruei Ayom

I'm living in Bor town in an area called Block 3.
Place of birth: Maar in Twic East County, Jonglei State.

Hobbies:
Reading newspapers.
Watching news and movies on TV.
Reading Bible.
Listening music and entertainments on radio.
Reading business tips and political affairs.

Family status:
I'm the first born son in the family, followed by my sister. We are two children only in our family. Both dad and mum died when we were very young and we were brought up by my maternal uncles up to now.

Education background:
I started my primary school in my village. The primary name is Maar primary school and finished my primary school in Bor B" primary school in 2009. I joined secondary school in Yei. At Nile High school Yei and completed it in 2013. I enrolled at Dr John Garang university of science and technology here in Bor one of the five public universities in the country where am pursuing a bachelor degree in business administration for four years in which I finished three years and Left with one year and will complete it this year if lockdown is lifted later this year.

Experience:
I worked for Radio Jonglei 95.9 FMone of the leading radios in Jonglei state for three years where I was a reporter and presenter as well.
I also worked for Voice of Reconciliation 98.4 FM as a reporter for seven months and quit for studies.
And to ATAKA HUB. where am still working now.
Then I recently attended ASKNET training online where I learned a lot of things including using Github application, how to write stories on Github, how to used tools for repairing phones, computers, radios, touches, lit lights and many more. The #ASKnet training helps me boost my knowledge in learning new things I never learned before.

I'm interesting in doing new things to help my community through ASKnet funded project if possible. I'm interesting in helping my community through the request made when we did community repair cafe that they need another three days for phones repairs and also to training them on business skills and empower them to start their own businesses.
